frontend/src/budget_simulation/BudgetSimulation.js

```diff
@@ -168,6 +256,22 @@ class Budget extends React.Component {
                     type={"submit"}
                     onClick={this.resetBudget}
                 > Reset </button>
+
+                <div>
+                    Sample size:
+                    <input
+                        type={"number"}
```

Comment:
Just FYI -- since all you're doing here is setting an attribute to a string or a number, there's no need for the brackets. They're only necessary if you're going to evaluate some JS. (so, `type="number"` is fine but you need them for `value={this.state.sampleSize}`)

It works anyway because React is evaluating the JS, which is just a string.